include/Reader/reader.h

```diff
@@ -3241,6 +3251,13 @@ class ReaderBase {
   virtual bool arrayGet(CORINFO_SIG_INFO *Sig, IRNode **RetVal) = 0;
   virtual bool arraySet(CORINFO_SIG_INFO *Sig) = 0;
 
+  /// \brief Check whether structs are represented by pointers on the operand
+  /// stack.
+  ///
+  /// \returns true iff structs are represented by pointers on the operand
+  /// stack.
+  virtual bool structsAreRepresentedByPointers() = 0;
```

Comment:
There is one place in ReaderBase (in storeObj) that needs this info and it didn't fell right to hard-code it to true there. GenIR has it hard-coded to true and never calls this method.